src/components/selection_input/selection_input.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, onWillUpdateProps, useEffect, useRef, useState } from "@odoo/owl";
```

Comment:
In commit message I'm pretty sure "redondant" isn't an english word. "redundant" ?